### PR TITLE
New presence: Champion.gg

### PR DESCRIPTION
**Pull request by drull1000**
initial page
matchup
runes
statistics

### CODE REVIEW

File: Championgg/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "service": "Championgg",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/4vqpAue.jpg",
+    "thumbnail": "https://i.imgur.com/td3BVmm.png",
+    "color": "#02442F",
+    "tags": [
+      "champion",
+      "championgg",
+      "league of legends",
```
Comment (Alanexei) [inline Championgg/dist/metadata.json:19]: Tags cannot have spaces in them.

File: Championgg/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "dist"
```
Comment (Alanexei) [inline Championgg/tsconfig.json:4]: Change this to `./dist`

Comment (drull1000) [reply Championgg/tsconfig.json:4]: Do you mean "outDir": "./dist" ?

Comment (drull1000) [reply Championgg/tsconfig.json]: done :D

**Pull request merged**